kernel/mmu.c

```diff
@@ -581,7 +581,7 @@ void *k_mem_map_impl(uintptr_t phys, size_t size, uint32_t flags, bool is_anon)
 	/* Need extra for the guard pages (before and after) which we
 	 * won't map.
 	 */
-	total_size = size + CONFIG_MMU_PAGE_SIZE * 2;
+	total_size = size + (CONFIG_MMU_PAGE_SIZE * 2);
```

Comment:
this is explicitly marked as exception in the rule description: https://wiki.sei.cmu.edu/confluence/display/c/EXP00-C.+Use+parentheses+for+precedence+of+operation

![x](https://github.com/zephyrproject-rtos/zephyr/assets/891546/e07bc05b-5970-4d23-919d-6e74db96e5b6)

it's the *literal* example.

Why are doing these?